frontend/services/entities/teams.ts

```diff
@@ -51,21 +52,15 @@ export default {
     return sendRequest("GET", path);
   },
   loadAll: ({
-    page = 0,
-    perPage = 20,
     globalFilter = "",
   }: ILoadTeamsParams = {}): Promise<ILoadTeamsResponse> => {
-    const { TEAMS } = endpoints;
-
-    // TODO: add this query param logic to client class
-    const pagination = `page=${page}&per_page=${perPage}`;
-
-    let searchQuery = "";
-    if (globalFilter !== "") {
-      searchQuery = `&query=${globalFilter}`;
-    }
+    const queryParams = {
+      query: globalFilter,
+    };
 
-    const path = `${TEAMS}?${pagination}${searchQuery}`;
+    const queryString = buildQueryStringFromParams(queryParams);
```

Comment:
`buildQueryStringFromParams()` is really coming in handy! Makes the code much more readable.